src/backend/utils/html/replace-entities.js

```diff
@@ -0,0 +1,21 @@
+const entities = require('entities');
+
+function decode(codeElement) {
+  // decode twice for double encoded entities
+  const result = entities.decodeHTML(codeElement.innerHTML);
+  return entities.decodeHTML(result);
+}
+
+module.exports = function (dom) {
+  if (!(dom && dom.window && dom.window.document)) {
```

Comment:
I wonder if it'd be possible to use the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here
```suggestion
  if (!(dom && dom.window?.document)) {
```

Reply:
I don't think we should introduce this, as it depends on node 14.  Wait until it's baked a bit longer and found its way into an LTS.

Reply:
:+1: